### PR TITLE
*: drop networkd section

### LINKED ISSUE #638: Drop networkd section from spec 3.0
# Feature Request #

## Environment ##

Any

## Desired Feature ##

If the outcome of https://github.com/coreos/fedora-coreos-tracker/issues/24 is to move away from networkd, we should consider dropping the `networkd` section from spec 3.0.

### PR DESCRIPTION
**Pull request by arithx**
Drop the networkd section from spec 3.0. networkd is not as widespread
as systemd and everything done in this section can be accomplished in
the files section.

Fixes #638

### CODE REVIEW

**Comment by ajeddeloh** [comment]
retest this please

**Comment by ajeddeloh** [comment]
retest this please

**Comment by ajeddeloh** [comment]
retest this please

File: doc/migrating-configs.md [hunk 1]
```diff
@@ -170,27 +170,6 @@ The `ignition` section has gained a new section named `security`, which can be u
 }
 ```
 
-### networkd dropins
```
Comment (bgilbert) [inline doc/migrating-configs.md]: I'm not sure we should be rewriting history like this.  Instead, the migration instructions to version 3 should explain converting networkd units/dropins to regular files.

Comment (arithx) [reply doc/migrating-configs.md]: Dropped this change and opened up #705

**Pull request merged**